src/register/RegistrationFields/NameField/constants.js

```diff
@@ -0,0 +1,5 @@
+export const INVALID_NAME_REGEX = /[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)?/gi; // eslint-disable-line no-useless-escape
+
+export const urlRegex = new RegExp(INVALID_NAME_REGEX);
+
+export const NAME_FIELD_LABEL = 'name';
```

Comment:
I am wondering if we need this level of division for constants. It feels like an overkill. In my opinion it would be easier and cleaner to have these directly defined in the validator file and the NameField file 

Same applies to other registration fields.